### DISCUSSION TITLE
Extensions shows a blank white window

### DISCUSSION THREAD

**Discussion by pjc2007** [lucono/karma-test-explorer#106] [Issues / Help]
I have recently noticed, that (disabling extensions until I found it is this one), that as soon as I enable it, I get and extra blank white window showing

![image](https://github.com/user-attachments/assets/a43bd2ea-dc57-4b3e-bd25-51600b7e8eba)

Not sure what may be causing this (I have uninstalled, reinstalled but it is still there)

Using a window finder utility, Winspector, it seems to be around all of vscode (over my 2 screens)...

![image](https://github.com/user-attachments/assets/6dc1be2a-355c-4fbe-b692-cf3dfc4a5db8)

![image](https://github.com/user-attachments/assets/96d43866-482d-4f15-be87-c14f0ce95742)

**Comment by Astrophizz** [comment]
I think it's related to a recent update of the "Test Adapter Converter" extension, which Karma Test Explorer depends on. That's where I ended up after an extension bisect and that extension just had a new release while Karma Test Explorer hasn't.

**Comment by pjc2007** [comment]
Thanks for that, I have logged the same thing at the extension you mention above (we also get it on another developers machine)

**Comment by Janneman96** [comment]
I have the same issue. I didn't notice your issue report at first.

**Comment by Janneman96** [comment]
To anyone fixing this; there's some more information from me in https://github.com/lucono/karma-test-explorer/issues/95.

**Comment by jackiems82** [comment]
I'm having the same issue...

**Comment by danisss9** [comment]
I tried downgrade test ui explorer, karma test explorer and the test adapter to the oldest version of it that still works with my tests

And it still shows the blank page as soon as it loads the tests.

So i think it probably was caused by a vs code update and not an extension update.
